packages/@lwc/engine/scripts/rollup/rollup.config.js

```diff
@@ -45,6 +48,7 @@ function rollupConfig({ format = 'es' } = {}) {
                 typescript,
                 include: TS_WHITELIST,
             }),
+            rollupLwcFeatures({ featureFlags }),
```

Comment:
We could also make the module id (`@lwc/features`) and runtime member expression (`global.LWC_config.features`) configurable.